src/dotnet/commands/dotnet-sln/LocalizableStrings.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DotNet.Tools.Sln
+{
+    internal class LocalizableStrings
+    {
+        public const string AppFullName = ".NET modify solution file(s) command";
```

Comment:
Should this be ".NET modify solution command"? It doesn't have the ability to modify multiple solutions in one command invocation right?